druid/src/win_handler.rs

```diff
@@ -40,6 +40,8 @@ use crate::{
 
 use crate::command::sys as sys_cmd;
 
+const RUN_COMMANDS_TOKEN: IdleToken = IdleToken::new(1);
```

Comment:
ugh I so wish there was a way to have a `const` function that incremented a global counter at compile time, so we could have `const fn next() -> IdleToken` and it would ensure that each token was new. I guess something like this is possible with a proc macro, but that feels like overkill.

Reply:
We can (ab)use rust macro : 
```rust
macro_rules! idle_id!() {
    {} => { uniq_hash(file!(), column!()) }
}
```
But I think it is not the best option either ;)